src/components/Transcript/Transcript.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import TanscriptSelector from './TranscriptMenu/TranscriptSelector';
+import { createTimestamp, timeToMs } from '@Services/utility-helpers';
+import './Transcript.scss';
+
+const Transcript = ({ transcripts }) => {
+  const [transcript, setTranscript] = React.useState([]);
+  const [transcriptTitle, setTranscriptTitle] = React.useState('');
+  const [transcriptUrl, setTranscriptUrl] = React.useState('');
+
+  // React refs array for each timed text value in the transcript
+  let textRefs = React.useRef([]);
+  const transcriptContainerRef = React.useRef();
+
+  React.useEffect(() => {
+    if (transcripts && transcripts.length > 0) {
```

Comment:
Might be able to short-circuit this evaluation with a `if (transcripts?.length > 0)...`

Reply:
Didn't know that javascript had that :100: